infra/docker-compose/docker-compose.online.yml

```diff
@@ -5,11 +5,16 @@ services:
     image: ${FEAST_SERVING_IMAGE}:${FEAST_VERSION}
     volumes:
       - ./serving/${FEAST_ONLINE_SERVING_CONFIG}:/etc/feast/application.yml
+      # Required if authentication is enabled on core and
+      # provider is 'google'. GOOGLE_APPLICATION_CREDENTIALS is used for connecting to core.
+      - ./gcp-service-accounts/${FEAST_BATCH_SERVING_GCP_SERVICE_ACCOUNT_KEY}:/etc/gcloud/service-accounts/key.json
```

Comment:
I really don't want to force users to use GCP in order to try out Feast. We already have that problem with historical serving. Is there a reason to set this as the default option? Will it fail if an account isnt provided?

Reply:
No, It doesn't fail if it isn't provided, If auth is enabled and set to `google`, then this will be used otherwise it is ignored.